mrp_lock_lot/i18n/fr.po

```diff
@@ -0,0 +1,153 @@
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 8.0\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2015-03-30 09:55+0000\n"
+"PO-Revision-Date: 2015-03-30 09:55+0000\n"
+"Last-Translator: <>\n"
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: \n"
+"Plural-Forms: \n"
+
+#: view:wiz.lock.lot:mrp_lock_lot.view_wiz_lock_lot_form
+#. module: mrp_lock_lot
+msgid "Cancel"
+msgstr "Annuler"
+
+#: field:product.category,lot_default_locked:0
+#. module: mrp_lock_lot
+msgid "Create lot in locked status"
+msgstr "Verrouiller les lots créés"
+
+#: field:wiz.lock.lot,create_uid:0
+#. module: mrp_lock_lot
+msgid "Created by"
+msgstr "Créé par"
+
+#: field:wiz.lock.lot,create_date:0
+#. module: mrp_lock_lot
+msgid "Created on"
+msgstr "Créé le"
+
+#: code:addons/mrp_lock_lot/models/mrp_production_lot.py:43
+#. module: mrp_lock_lot
+#, python-format
+msgid "Error!: Found stock movements for lot: \"%\" with location destination type in virtual/company"
+msgstr "Erreur ! : Il existe des réservations de stock pour le lot \"%s\"."
+
+#: field:wiz.lock.lot,id:0
+#. module: mrp_lock_lot
+msgid "ID"
+msgstr "Id."
+
+#: help:product.category,lot_default_locked:0
+#. module: mrp_lock_lot
+msgid "If checked, production lots will be created locked by default"
+msgstr "Si coché, les lots de production seront créés verrouillés par défaut"
+
+#: field:wiz.lock.lot,write_uid:0
+#. module: mrp_lock_lot
+msgid "Last Updated by"
+msgstr "Mis à jour par"
+
+#: field:wiz.lock.lot,write_date:0
+#. module: mrp_lock_lot
+msgid "Last Updated on"
+msgstr "Mis à jour le"
+
+#: view:stock.production.lot:mrp_lock_lot.view_production_lot_form_inh_locklot
+#. module: mrp_lock_lot
+msgid "Lock"
+msgstr "Verrouiller"
+
+#: view:product.category:mrp_lock_lot.product_category_form_view_inh_locklot
+#. module: mrp_lock_lot
+msgid "Lock Lot Properties"
+msgstr "Propriétés de verrouillage des lots"
+
+#: view:wiz.lock.lot:mrp_lock_lot.view_wiz_lock_lot_form
+#. module: mrp_lock_lot
+msgid "Lock lots"
+msgstr "Verrouiller les lots"
+
+#: model:ir.actions.act_window,name:mrp_lock_lot.action_lock_lot
+#. module: mrp_lock_lot
+msgid "Lock/Unlock lot"
+msgstr "Verrouiller/déverrouiller le lot"
+
+#: view:wiz.lock.lot:mrp_lock_lot.view_wiz_lock_lot_form
+#. module: mrp_lock_lot
+msgid "Lock/Unlock lots"
+msgstr "Verrouiller/déverrouiller les lots"
+
+#: view:stock.production.lot:mrp_lock_lot.search_product_lot_filter_inh_locklot
+#: field:stock.production.lot,locked:0
+#: view:stock.quant:mrp_lock_lot.quant_search_view_inh_locklot
+#: field:stock.quant,locked:0
+#. module: mrp_lock_lot
+msgid "Locked"
+msgstr "Verrouillé"
+
+#: view:stock.quant:mrp_lock_lot.quant_search_view_inh_locklot
+#. module: mrp_lock_lot
+msgid "Lot"
+msgstr "Lot"
+
+#: model:mail.message.subtype,description:mrp_lock_lot.mt_lock_lot
+#: model:mail.message.subtype,name:mrp_lock_lot.mt_lock_lot
+#. module: mrp_lock_lot
+msgid "Lot Locked"
+msgstr ""
```

Comment:
Sure, must have missed it. I'll make another proposal with improved strings in a few minutes if you don't mind.